tests/v2/test_message_store_sqlite.nim

```diff
@@ -25,13 +25,16 @@ const
 proc newTestDatabase(): SqliteDatabase =
   SqliteDatabase.init("", inMemory = true).tryGet()
 
+proc now(): Timestamp =
+  getNanosecondTime(getTime().toUnixFloat())
+
 proc getTestTimestamp(offset=0): Timestamp = 
-  Timestamp(getNanosecondTime(epochTime()))
+  Timestamp(getNanosecondTime(getTime().toUnixFloat() + offset.float))
```

Comment:
Curious why this replacement. I remember it was recommended to use `getTime()` instead of `epochTime()`, but thought maybe you had a different reason?

Reply:
From nim documentation for `epochTime()`:

> Gets time after the UNIX epoch (1970) **in seconds**. It is a float because sub-second resolution is likely to be supported (depending on the hardware/OS).


> **`getTime` should generally be preferred over this proc.**

We want sub-second precision which is only provided by `getTime()` as I have found out empirically.